src/reactor/global.rs

```diff
@@ -49,6 +49,9 @@ impl Drop for HelperThread {
 
 fn run(mut reactor: Reactor, done: Arc<AtomicBool>) {
     while !done.load(Ordering::SeqCst) {
-        reactor.turn(None);
+        match reactor.turn(None) {
```

Comment:
Could this use `panic` as it's not going to take down any other thread and otherwise it seems bad to me to silently cover up this failure by accident.

Reply:
Done